netebpfext/net_ebpf_ext_bind.c

```diff
@@ -0,0 +1,249 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+/**
+ * @brief    This file implements the BIND program type hook on eBPF for Windows.
```

Comment:
```suggestion
 * @file This file implements the BIND program type hook on eBPF for Windows.
```
Use `@file` for file docs per https://www.doxygen.nl/manual/commands.html#cmdfile

Reply:
Fixed in #520